connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaOffsetBackingStoreTest.java

```diff
@@ -105,34 +112,53 @@ public class KafkaOffsetBackingStoreTest {
 
     @Mock
     KafkaBasedLog<byte[], byte[]> storeLog;
+    @Mock
+    Supplier<TopicAdmin> topicAdminSupplier;
+    @Spy
+    @InjectMocks
     private KafkaOffsetBackingStore store;
 
-    private Capture<String> capturedTopic = EasyMock.newCapture();
-    private Capture<Map<String, Object>> capturedProducerProps = EasyMock.newCapture();
-    private Capture<Map<String, Object>> capturedConsumerProps = EasyMock.newCapture();
-    private Capture<Supplier<TopicAdmin>> capturedAdminSupplier = EasyMock.newCapture();
-    private Capture<NewTopic> capturedNewTopic = EasyMock.newCapture();
-    private Capture<Callback<ConsumerRecord<byte[], byte[]>>> capturedConsumedCallback = EasyMock.newCapture();
+    private MockedStatic<ConnectUtils> connectUtilsMockedStatic;
+
+    private final ArgumentCaptor<String> capturedTopic = ArgumentCaptor.forClass(String.class);
+    @SuppressWarnings("unchecked")
+    private final ArgumentCaptor<Map<String, Object>> capturedProducerProps = ArgumentCaptor.forClass(Map.class);
```

Comment:
We can use the `@Captor` annotation here and elsewhere to simplify things a bit:
```suggestion
    @Captor
    private ArgumentCaptor<Map<String, Object>> capturedProducerProps;
```

Reply:
I agree, I have used `@Captor` in the next version.

Reply:
However, I left some of the more obscure ArgumentCaptors defined inline. If you disagree with that I will hoist them as fields as well.